test-lit/semantics/if_stmt02.f90

```diff
@@ -0,0 +1,6 @@
+!ERROR: IF statement is not allowed in IF statement
```

Comment:
Why did this come as the first line here (and not the run line)?
Why doesn't git show this file as a renaming/movement of file (test/semantics/if_stmt02.f90)? At the end of this patch the file is shown as deleted.

Reply:
My script expects a license boilerplate or other comments at the beginning, I'll change it to detect if the insert line is after prefixes.

It has been renamed/moved, git just doesn't recognise it. It's happened to a few files, and I verified all of them. It has the same effect so it doesn't matter too much.